### PR TITLE
React week1 homework

### PR DESCRIPTION
**Pull request by AlejandroGispert**
In the space travelling website. I created a basic project, with some CSS and using the basic React + next.js setup. I constructed the new sections as asked in the homework. I will be improving it throughout the week. Here is a screenshot.
Screenshot 2024-08-26 at 14 42 05

### CODE REVIEW

File: app/layout.js [hunk 1]
```diff
@@ -19,10 +19,10 @@ export const RootLayout = ({ children }) => {
         {children}
         {/* TASK - React 1 week 1 */}
```
Comment (ddobby94) [inline app/layout.js]: Feel free to delete this comment

File: app/about_us/page.js [hunk 2]
```diff
@@ -45,18 +10,19 @@ export const Crew = () => {
...
+          <OurValues />
         </section>
         <section className="card">
           <h2>The crew</h2>
-          <OurCrew/>
+          <OurCrew />
+        </section>
+        <section className="card">
+          <h2>Partners</h2>
+          <OurPartners />
```
Comment (ddobby94) [inline app/about_us/page.js:21]: Great job! This is how it should be done in React :)

File: app/about_us/page.js [hunk 3]
```diff
@@ -1,42 +1,7 @@
...
-  // Create the "Our Crew section"
-  // Use the descriptions provided in /app/about_us/README.md
-  // Use the pictures from /public/crew
-  // Some inspiration ideas found in /data/inspiration_about_us
-  return (
-    <p> ADD OUR Partners HERE </p>
-  );
-}
-
+import styles from "./page.module.css";
```
Comment (ddobby94) [inline app/about_us/page.js]: One super small thing:

We like to keep the imports in order. And the `.css` file is usually the last one.

File: app/about_us/OurValues.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import styles from "./page.module.css";
+export default function OurValues() {
+  return (
+    <>
+      <ol>
+        <li>
+          <h3>Exploration: </h3>{" "}
```
Comment (ddobby94) [inline app/about_us/OurValues.js]: If you need some extra space after the header it is better to use some css 😉 
You can create a class for the `<ol className={styles.ourValuesContainer}>` and use that class to apply some margins/paddings for the children.

File: app/about_us/OurValues.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import styles from "./page.module.css";
+export default function OurValues() {
+  return (
+    <>
```
Comment (ddobby94) [inline app/about_us/OurValues.js]: No need for this empty container here, you can just use:
```
return (
  <ol>
  ...
  </ol>
)
```

File: app/about_us/OurCrew.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+import image4 from "../../public/crew/image-victor-glover.png";
+import image5 from "../../public/crew/nasaCrewMember1.jpg";
+import image6 from "../../public/crew/nasaCrewMember2.webp";
+import Image from "next/image";
+import HanaLeePic from "../../public/crew/astronautHanaLee.webp";
+import PatelPic from "../../public/crew/astronautPatel.webp";
+export default function OurCrew() {
+  return (
+    <>
+      <h4 style={{ margin: "60px" }}>
```
Comment (ddobby94) [inline app/about_us/OurCrew.js]: Don't get lazy here 😛 
It is usually forbidden to have in line styles with react.
They are very difficult to identify and maintain in the long run!
So please move the css out to the `.css` file.

**Review by ddobby94 — COMMENTED** [review]
Great job!
It seems like you are getting the basics of React, which is the most important part here.
I've left some smaller comments to make your code nicer, but otherwise it looks nice :)

File: app/about_us/OurCrew.js (hunk 6 above)
Comment (AlejandroGispert) [reply app/about_us/OurCrew.js]: yes. i fix. i was just testing it worked the double curly brackets

**Pull request closed without merging**